docs/administration/data/octopus-database/index.md

```diff
@@ -18,6 +18,10 @@ You are responsible for the routine maintenance of your Octopus database. Perfor
 
 You are responsible for taking database backups and testing your disaster recovery plans. Refer to [Backup and restore](/docs/administration/data/backup-and-restore.md) for more information about backing up Octopus Deploy and recovering from failure.
 
+### High Availabilty Databases {#highavailability}
+
+Octopus Server does not support running against a Sql database with Database Mirroring or Sql Replication enabled. If you require a highly-available database solution, [Always On Availability Groups](https://docs.microsoft.com/en-us/sql/database-engine/availability-groups/windows/overview-of-always-on-availability-groups-sql-server?view=sql-server-2017) are recommended.  
```

Comment:
Putting myself in the shoes of the customer, it feels a bit funny to start with "no HA for you". I feel like this section would be more natural starting with "If you want to configure your database for high availability, awesome, here's our recommendation. Oh, BTW, don't use these techniques, they aren't tested nor supported." :)

Reply:
reordered the wording so the recommendation was upfront